package.json

```diff
@@ -33,6 +34,8 @@
     "PlatformConstants.web.js",
     "react-native-gesture-handler.d.ts",
     "README.md",
+    "RNGestureHandlerModule.js",
+    "jestConfig.js",
```

Comment:
This should be `jestSetup.js`, I think? Which probably explains why the `jestSetup.js` appears to be missing from the 1.2.0 release.